### PR TITLE
Design improvements

### PR DESCRIPTION
**Pull request by henryhadlow**
Small design improvements. See commit history or https://www.pivotaltracker.com/story/show/98274290

### CODE REVIEW

File: app/assets/stylesheets/govuk_elements/_forms.scss [hunk 1]
```diff
@@ -83,10 +83,6 @@ fieldset {
   display: block;
   color: $secondary-text-colour;
   margin-bottom: 5px;
-
```
Comment (alexander-bobin) [inline app/assets/stylesheets/govuk_elements/_forms.scss]: Better to remove/overwrite this in our `forms.scss`

File: app/assets/stylesheets/govuk_elements/forms/_form-block-labels.scss [hunk 2]
```diff
@@ -9,13 +9,14 @@
 .block-label {
 
   display: block;
+  box-sizing: border-box;
```
Comment (alexander-bobin) [inline app/assets/stylesheets/govuk_elements/forms/_form-block-labels.scss]: Better to remove/overwrite this in our forms.scss

File: app/assets/stylesheets/govuk_frontend_toolkit/_typography.scss [hunk 3]
```diff
@@ -3,6 +3,8 @@
 @import '_device-pixels.scss';
 @import '_url-helpers.scss';
 
+$text-indent: 50px;
```
Comment (alexander-bobin) [inline app/assets/stylesheets/govuk_frontend_toolkit/_typography.scss]: This should live in our `typography.scss`

Comment (henryhadlow) [inline app/assets/stylesheets/govuk_frontend_toolkit/_typography.scss]: Agreed, except our typography file isn't high enough in the stack. Could just put it in `site.scss`?

File: app/assets/stylesheets/petitions/_forms.scss [hunk 4]
```diff
@@ -38,16 +43,13 @@ select.form-control {
 
 .character-count {
   color: $secondary-text-colour;
-  text-align: right;
+  float: right;
```
Comment (alexander-bobin) [inline app/assets/stylesheets/petitions/_forms.scss]: This makes it look pretty ugly on mobile without JS. We OK with that?

screen shot 2015-07-07 at 11 15 09

Comment (henryhadlow) [inline app/assets/stylesheets/petitions/_forms.scss]: Yeah I reckon so. If mobile devices have average js support, around 1% of users will see that. It's functional.

File: app/assets/stylesheets/petitions/views/_shared.scss [hunk 5]
```diff
@@ -183,7 +188,25 @@ input.back-page {
     }
   }
 }
-
+.about-item-count-response, .about-item-count-debate {
+  background-repeat: no-repeat;
+}
+.about-item-count-response {
+  background-position: 0 26px;
+  background-image: image-url("graphics/graphic_crown.png");
```
Comment (alexander-bobin) [inline app/assets/stylesheets/petitions/views/_shared.scss]: We can use the background-image mixin inside `helpers.scss` for this.

File: app/views/petitions/_open_petition_show.html.erb [hunk 6]
```diff
@@ -18,13 +18,15 @@
...
     <span class="signature-count-current" style="width: <%= signatures_threshold_percentage(petition) %>"></span>
   </div>
-  <p class="signature-count-goal">
-    <% if petition.signature_count < Site.threshold_for_response %>
-      <%= number_with_delimiter(Site.threshold_for_response) %>
-    <% else %>
-      <%= number_with_delimiter(Site.threshold_for_debate) %>
-    <% end %>
-  </p>
+  <% unless petition.debate_threshold_reached_at? || petition.scheduled_debate_date? || petition.debate_outcome_at? %>
```
Comment (alexander-bobin) [inline app/views/petitions/_open_petition_show.html.erb]: Do we have an integration tests for this?

File: app/views/sponsor_mailer/petition_and_email_confirmation_for_sponsor.html.erb [hunk 7]
```diff
@@ -4,7 +4,7 @@
 
 <hr />
 
-<p><%= @petition.creator_signature.name %>’s petition:</p>
+<p><%= @petition.creator_signature.name %>'s petition:</p>
```
Comment (alexander-bobin) [inline app/views/sponsor_mailer/petition_and_email_confirmation_for_sponsor.html.erb]: Good catch!

File: app/views/petitions/_open_petition_show.html.erb (hunk 6 above)
Comment (henryhadlow) [inline app/views/petitions/_open_petition_show.html.erb]: We have a couple of tests, but not enough to cover all the scenarios. Probably outside of my XP level

**Pull request merged**